cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/store/DefaultMetadataStore.java

```diff
@@ -464,12 +465,18 @@ public void apply(MetadataDataset input) throws Exception {
   }
 
   @Override
-  public Set<MetadataRecord> getSnapshotBeforeTime(MetadataScope scope, final Set<Id.NamespacedId> entityIds,
-                                                    final long timeMillis) {
+  public Set<MetadataRecord> getSnapshotBeforeTime(final MetadataScope scope, final Set<Id.NamespacedId> entityIds,
```

Comment:
shouldn't be necessary once https://github.com/caskdata/cdap/pull/4627/files#r45781784 is done